.github/workflows/test.yml

```diff
@@ -41,6 +41,12 @@ jobs:
             elixir: 1.10.4
```

Comment:
I think we can remove testing on OTP 22.

Reply:
removed.
3756b85631b885dfcecbf8ba5fd270c29a8d635c